snapcraft.yaml

```diff
@@ -16,13 +16,13 @@ plugs:
     read:
       - $HOME/.dss/config
 apps:
-  dss:
+  data-science-stack:
```

Comment:
Actually, I think the alias request you had approved will require to app to remain "dss".  The alias for dss to data-science-stack.dss will require the app to be named dss.  I think you just need to revert this one line for it to work as expected with the alias.